doc/nrf/libraries/bluetooth_services/mesh/vnd/dm_srv.rst

```diff
@@ -0,0 +1,50 @@
+.. _bt_mesh_dm_srv_readme:
+
+Distance Measurement Server
+###########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Distance Measurement Server vendor model provides capabilities to measure distance between Bluetooth® mesh devices within radio proximity.
+The measurements are conducted through the :ref:`mod_dm` library.
+
+At creation, each Distance Measurement Server instance must be initialized with a memory object that can hold one or more measurement results.
+The Distance Measurement Server populates this memory object whenever a new measurement is conducted.
+The results can be queried by a :ref:`bt_mesh_dm_cli_readme`.
+
+.. note::
+   The current implementation is :ref:`experimental <software_maturity>`.
+
+States
+======
+
+None
+
+Extended models
+===============
+
+None
+
+Persistent storage
+==================
+
+The Distance Measurement Server has following runtime configuration options:
+
+* Default time to live for sync messages.
+* Default timeout for distance measurement procedure.
+* Default reflector start delay.
```

Comment:
Is the word "Default" needed in these 3 lines.

Reply:
I added it to make the user aware that it is the default measurment_start values because it is possible to provide custom parameters for these values in the measurment_start message.